### PR TITLE
RzIL tree view

### LINKED ISSUE #1133: RzIL and C AST visual representation
Like it is done for Binary Ninja IL https://github.com/withzombies/bnil-graph

- [x] RzIL graph (https://github.com/rizinorg/cutter/pull/3191)
- [ ] C AST graph

image

From https://twitter.com/withzombies/status/1087521995577876481

### PR DESCRIPTION
**Pull request by wargio**
**Your checklist for this pull request**
- [ ] I've read the [guidelines for contributing](https://cutter.re/docs/contributing/code/getting-started.html) to this repository
- [ ] I made sure to follow the project's [coding style](https://cutter.re/docs/contributing/code/development-guidelines.html)
- [ ] I've updated the [documentation](https://cutter.re/docs/user-docs.html) with the relevant information (if needed)

**Detailed description**

Adds the RzIL tree view.

image

Partially handles #1133

### CODE REVIEW

File: src/core/Cutter.cpp [hunk 1]
```diff
@@ -3543,25 +3533,27 @@ QList<BinClassDescription> CutterCore::getAllClassesFromBin()
...
     RzBinSymbol *sym;
-    RzBinField *f;
+    RzBinClassField *f;
+    size_t index = 0;
     CutterRzListForeach (cs, iter, RzBinClass, c) {
         BinClassDescription classDescription;
         classDescription.name = c->name;
         classDescription.addr = c->addr;
-        classDescription.index = c->index;
+        classDescription.index = index;
```
Comment (thestr4ng3r) [inline src/core/Cutter.cpp]: I think you can just remove the index field from BinClassDescription, it does not make much sense imo.

**Comment by XVilka** [comment]
Broken windows build (jsdec)

**Comment by wargio** [comment]
There seems to be a mess with the plugin install paths on the windows release (and i would not be surprised if also linux and osx has the same issue)
@thestr4ng3r @XVilka @ret2libc

**Comment by XVilka** [comment]
An eternal problem.

File: src/core/Cutter.cpp (hunk 1 above)
Comment (wargio) [reply src/core/Cutter.cpp]: removed here. https://github.com/rizinorg/cutter/pull/3194/commits/b817ea67006a5b47846b3f81377bd1e2022251a2

**Comment by wargio** [comment]
i have splitted this PR. before i want to fix the build, then i can add this.

**Comment by wargio** [comment]
@XVilka @thestr4ng3r now you can approve it :D

**Review by XVilka — APPROVED** [review]

**Pull request merged**